overlore/townhall/service.py

```diff
@@ -48,27 +47,38 @@ async def start() -> None:
     if TORII_WS is None:
         raise RuntimeError("Failure to provide WS url")
 
-    db = DatabaseHandler.instance().init()
+    DatabaseHandler.instance().init()
```

Comment:
some changes were done by the linter in response to me commenting out code, in particular the removal of assignments